src/main/scala/magellan/BoundingBox.scala

```diff
@@ -112,14 +112,18 @@ case class BoundingBox(xmin: Double, ymin: Double, xmax: Double, ymax: Double) {
       Line(Point(xmin, ymax), Point(xmax, ymin))
     )
 
+    if (contains(shape.boundingBox)) {
+      return Contains
```

Comment:
somewhat related, but am I correct that this Relation is ignored by the shortcut?